test/Tool/range12.opt

```diff
@@ -0,0 +1,11 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: %FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i8 = var (knownBits=000000xx) (nonNegative) (nonZero) (signBits=6) (range=[1,3))
```

Comment:
can you explain the purpose of providing all of these dataflow facts? are they necessary to make the test case work? if not, you should remove the extra ones as this is confusing and also provides souper with redundant information that makes it hard to figure out what effect the range information is having